core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -219,11 +221,13 @@ var ExplorationEditorTranslationTab = function() {
     await waitFor.elementToBeClickable(
       deleteRecordButton,
       'Delete Record button is not clickable');
```

Comment:
Do we still need these checks? (I think the action.click button does this before clicking the button)

Ditto for all other places*

Reply:
Ya sure I have removed most of them will remove remaining also